### DISCUSSION TITLE
Customize the page title instead of "Nuxt.js: Loading app... "

### DISCUSSION THREAD

**Discussion by uzunvictor** [nuxt/nuxt#28352] [Questions (Nuxt 2)]
Hi, can anyone help me with setting up the page title while the application is loading after the build?
In production the crawlers detect this title and that is not good for the seo.
Thanks.
![nuxtls_loading_app](https://github.com/user-attachments/assets/d1e731c6-c4f1-4ead-b6ac-e0fbc93672eb)

**Comment by xjccc** [comment]
You could use [useHead](https://nuxt.com/docs/api/composables/use-head)

**Reply by uzunvictor** [reply]
Thanks) but it's Nuxt 2

**Reply by xjccc** [reply]
Set a dynamic title?
```
// id.vue
 data() {
    return {
        title: ''
    };
  },
  head() {
    return {
      title:  this.title
    }
   }
```
